packages/data-api/package.json

```diff
@@ -20,7 +20,8 @@
     "uninstall-mv-refresh": "scripts/uninstallMvRefreshModule.sh",
     "build-analytics-table": "scripts/buildAnalyticsMaterializedView.sh",
     "drop-analytics-table": "scripts/dropAnalyticsMaterializedView.sh",
-    "refresh-analytics-table": "scripts/refreshAnalyticsTable.sh"
+    "refresh-analytics-table": "scripts/refreshAnalyticsTable.sh",
+    "refresh-analytics-table:full": "scripts/refreshAnalyticsTable.sh --full"
```

Comment:
Is there a specific reason to add this rather than just use `yarn workspace @tupaia/data-api refresh-analytics-table --full`?

Reply:
Just for discoverability, you would have to look inside the script to know that is an option.

Reply:
@rohan-bes thoughts?

Reply:
Oh sorry, ummm I suppose I'm not too fussed 👍 There's documentation in the README of the package explaining the different options, but that's a bit out of the way and doesn't outline them in detail. I'm cool with this 😄

Reply:
Ah it would help if I read it! 😆